Dockerfile

```diff
@@ -16,11 +14,7 @@ COPY main.go main.go
 COPY controllers/ controllers/
 
 # Build
-# the GOARCH has not a default value to allow the binary be built according to the host where the command
-# was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
-# the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
-# by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
-RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
Are we sure we want to remove the different arch building ?
we might need it in the future (but i guess we can add it back worst case)

Reply:
If we will need it, we can add it.
[kubemacpool](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/main/Makefile#L93) has it hardcoded as well.